akka-persistence/src/test/scala/akka/persistence/journal/SteppingInmemJournal.scala

```diff
@@ -0,0 +1,136 @@
+package akka.persistence.journal
+
+import akka.actor.ActorRef
+import akka.persistence.journal.inmem.InmemJournal
+import akka.persistence.{ AtomicWrite, PersistentRepr }
+import com.typesafe.config.{ ConfigFactory, Config }
+
+import scala.collection.immutable.Seq
+import scala.concurrent.{ Future, Promise }
+import scala.util.Try
+
+object SteppingInmemJournal {
+
+  /** allow the journal to do one operation */
+  case object Token
+  case object TokenConsumed
+
+  def config(instanceId: String): Config =
+    ConfigFactory.parseString(
+      s"""
+        |akka.persistence.journal.stepping-inmem.class=akka.persistence.journal.SteppingInmemJournal
```

Comment:
get the classname via `${classOf[SteppingInmemJournal].getName}`? It's more reliable for refactorings